RELEASE-NOTES.md

```diff
@@ -22,6 +22,15 @@
 ## New additions
 * Added support for `title` field in Streamlit definition in `snowflake.yml` project file.
 * Added `--auto-compress` flag to `snow stage copy` command enabling use of gzip to compress files during upload.
+* Added support for `--env` command line arguments for templating.
```

Comment:
Can I override streamlit name? Probably by doing something like `--env="streamlit.name=foo"`?

Reply:
no. Overrides only apply to env section.